src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAO.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.implicitConversions
 
 class GraphWorkspaceDAO extends WorkspaceDAO with GraphDAO {
 
-  override def save(workspace: Workspace, txn: RawlsTransaction): Workspace = txn withGraph { db =>
+  override def save(workspace: Workspace, txn: RawlsTransaction): WorkspaceContext = txn withGraph { db =>
```

Comment:
yeah you are right, the context is already used all over the place anyway in the DAOs so this was already a problem.  I guess its a wash on this as you said, and I'm ok with it, except the return type.  

We should reevaluate our DAOs at some point to decide whether we want to fix the DAOs to be implementation agnostic or keep letting them slide as they have.